docs/dictionary/command/send.lcdoc

```diff
@@ -4,7 +4,7 @@ Type: command
 
 Syntax: send <message> [ to <object> [in <time> [seconds | ticks | milliseconds]] ]
```

Comment:
Currently the message and object parameters here are not distinct from the names of existing dictionary entries, which means that the links will not work correctly. I assume that's why you removed the indefinite articles from the summary below. 

The best way to fix this would be to change the syntax to 
```
send <messageName> [ to <targetObject> [in <time> [seconds | ticks | milliseconds]] ]
```
and then make sure all the instances of `<message>` and `<object>` later on in the entry that are supposed to refer to the actual parameter rather than link to the general message and object dictionary entries are changed to `<messageName>` and `<targetObject>`.